UnitTestFrameworkPkg/UnitTestFrameworkPkg.dec

```diff
@@ -21,9 +21,12 @@
   Library/GoogleTestLib/googletest/googletest/include
   Library/GoogleTestLib/googletest/googlemock/include
   Library/SubhookLib/subhook
+  # MU_CHANGE - With great power comes great responsibility.
+  # Some Mu test collateral expects access to the framework internals.

```

Comment:
What is this "some" test that needs great responsibility in `PrivateInclude`?

Reply:
@os-d do you have more context on this?

Reply:
Nope, this predates me. I would check and see if the test still exists that expects to use something in PrivateInclude (i.e. remove this change and see if it still builds)

Reply:
Removed, not needed